### ISSUE TITLE
Few suggestion

### ISSUE THREAD

**Issue by jimmywarting** [PokemonGoF/PokemonGo-Bot-Desktop#104]
- [ ] Show lured pokestop on the map
- [ ] Click to walk somewhere (Good if you find a place with many lured)
- [ ] Make a walking path by drawing/clicking
  - [ ] A way to loop the path,
  - [ ] Go reverse (back and forth) 
- [ ] Mark spined pokestop (make them purple) (maybe show a time left inside info bubble)
- [ ] Show the name of the pokestop in the info box (or at least show where you are heading) the path you have walked is red so the path from your point to the next place could be yellow or blue
- [ ] Able to recycle items from the inventory
- [ ] Turn on Lucky egg from the inventory
- [ ] Turn on Incense from the inventory (once the bot can catch those pokemon as well)
- [ ] Click on revive or potion to give it to a pokemon who needs it
- [ ] Evolve from the pokemon tab
- [ ] able to configure the recycle config, I'm getting this:

> Not moving to any forts as there aren't enough space. You might want to change your config to recycle more items if this message appears consistently.
- [ ] able to configure catch/release config
- [ ] Give Pokémon nickname
- [ ] Able to toggle Pokémon as favorite
- [ ] Show max possible cp under the pokemon tab

**Comment by Outpox** [comment]
Many of this feature are not yet implemented in the bot iirc, so we'll have to wait.
Altough, showing the lured forts should be already doable.

**Comment by jimmywarting** [comment]
Mark spined pokestop should be doable also?

**Comment by Outpox** [comment]
I was wondering the same, but it's not the case in the webapp of the bot so I don't really know.

**Comment by tnsbm** [comment]
https://github.com/PokemonGoF/PokemonGo-Bot-Desktop/issues/138

**Comment by ProjectBarks** [comment]
# Planned

_From your list here are things I plan to accomplish for our app._
- Show lured pokestop on the map
- Make a walking path by drawing/clicking
  - A way to loop the path,
- Show the name of the pokestop in the info box (or at least show where you are heading) the path you have walked is red so the path from your point to the next place could be yellow or blue
- Able to recycle items from the inventory
- Evolve from the pokemon tab
- able to configure the recycle config, I'm getting this:
- able to configure catch/release config
- Give Pokémon nickname
- Show max possible cp under the pokemon tab

**Comment by EvanTheTerrible** [comment]
As it is in 0.0.2 Alpha, it would be nice to give people explicit instructions as to where the config files are located. If they've been using the other releases, I would hope they would be able to edit the config.json without screwing the whole thing up. Took me a few minutes to figure out where the file was and I'd like to save people the time searching. :-) Thanks for all your hard work so far.

For anyone wondering, the config.json file (on OSX atleast) is editable by showing the package contents of Pikabot.app, then navigating to Contents/Resources/app/gofbot/configs/config.json

**Comment by jcctr** [comment]
Is it me or the build version lacks some of the features I had when installing from the terminal ? I can't see the candies anymore, nor the A/D/S stats :/ 
Thanks for this amazing work anyway ;)

**Comment by Outpox** [comment]
Yep the builds lacks some commits which added those interesting features.

**Comment by EvanTheTerrible** [comment]
I'm looking (and learning) at the code to see how they were done in the terminal version to see if I can add them back to my version. I'm sure the folks working on this have better things to do. :-)

**Comment by Outpox** [comment]
`git checkout 98226a9` will get you to the oldest working version with the most features.
